### DISCUSSION TITLE
Rebuilding Forward Sensor

### DISCUSSION THREAD

**Discussion by kl3ss** [Security-Onion-Solutions/securityonion#7931] [General] [answered]
Hey there,

One of our forward sensors in our Deployment has bit the dust, so I will be looking at rebuilding it. The machine will be given the same name and have the same IP Address. My question is, is there anything I have to do / edit / remove on the Master Server so during the installation process it does not error out when trying to register with the master server?

Cheers

kl3ss

**Comment by thebee64** [comment, marked as answer]
Having been there not more than a few weeks ago, the one issue I ran into was removing the old forward sensor, before standing up the new one.  In the Docs under "Tips and Tricks" follow the steps to remove the old forward node from the Manager node, including the salt key and sensorstab.sls file.  This should clean things up for the new forward node..

https://docs.securityonion.net/en/2.3/removing-a-node.html

Rick
